cocos/ui/UIListView.cpp

```diff
@@ -93,7 +93,7 @@ void ListView::updateInnerContainerSize()
         case Direction::VERTICAL:
         {
             size_t length = _items.size();
-            float totalHeight = (length - 1) * _itemsMargin;
+            float totalHeight = ((float)length - 1) * _itemsMargin;
```

Comment:
length is unsigned int , if length is 0, (length - 1) will be a max value, so if _itemsMargin is not 0,totalHeight will be a large value
![1](https://cloud.githubusercontent.com/assets/2168829/7195997/eb15c6ee-e4fb-11e4-9e03-21b1b2c86833.png)
.